### PR TITLE
Update readme to not allow password for all linux VMs and adjust IP to list for dbnodes

### PR DESCRIPTION
**Pull request by rukawata**
This is to address #362 and reflect changes in #364

### CODE REVIEW

File: deploy/v2/README.MD [hunk 1]
```diff
@@ -216,7 +215,8 @@ A typical deployment lifecycle will require the following steps:
...
-| <sub></sub>                                                     | <sub>private_ip</sub>                         | <sub>string</sub> | <sub>(Optional) Static IP Address for jumpbox VM. Will be assigned   according to its order under jumpboxes if left empty.</sub>                                                                                                                                                                                                                                                                                       |
+| <sub></sub>                                                     | <sub>admin_nic_ips</sub>                         | <sub>list</sub> | <sub>(Optional) List of up to 2 static IP Addresses of HANA database VM for traffic from jumpboxes (1 IP for single HANA deployment and 2 IPs for HA pair). Will be assigned   according to its order under jumpboxes if left empty.</sub>                                                                                                                                                                                                                                                                                       |
```
Comment (persiaAziz) [inline deploy/v2/README.MD]: Is this intended? Asking because your PR title only talks about auth options for linux jb

Comment (rukawata) [reply deploy/v2/README.MD]: Sorry, my mistake. This is intended for all Linux VMs. Will correct that.

Comment (pabowers) [inline deploy/v2/README.MD]: Why is there a large space between "assigned" and "according"?

Comment (rukawata) [reply deploy/v2/README.MD]: Also, we changed the way we pass HANA server IP in this PR. I will add that to the PR description.

Comment (rukawata) [reply deploy/v2/README.MD]: Just removed that. Thanks.

**Pull request merged**